README.md

```diff
@@ -16,15 +16,14 @@ You can find more information about the process in the [contributing section](ht
 
 ## Who is fostering this initiative?
 
-Bitergia, as neutral actor in the software development analytics field aims at
-sharing the knowledge acquire during the last years. This book is currently focused on its main areas of expertise such as the usual metrics and KPI's to use,
+We thank [Bitergia] for starting this book and seeding it with its main areas of expertise such as the usual metrics and KPI's to use,
```

Comment:
```suggestion
We thank [Bitergia] (especially José Manrique López and Daniel Izquierdo) for starting this book and seeding it with its main areas of expertise such as the usual metrics and KPI's to use,
```

Suggesting to give specific props to these two, who wrote most of the initial book it seems.

Reply:
Great idea!